peps/pep-0731.rst

```diff
@@ -101,7 +102,7 @@ The members of the working group are:
 - Petr Viktorin
 - Victor Stinner
 - Steve Dower
-- Irit Katriel
+- Erlend Aasland
```

Comment:
We should maybe sort the list by family name, ex: https://peps.python.org/pep-8104/#candidates

Reply:
I would default to lexicographic sort by the full name, it avoids any difficulties around which name is the family name (e.g. some Asian names), or what to do for people who don't have/use a family name.

Compare: https://peps.python.org/#authors-owners

Compulsory link: https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

Reply:
Sure; are the PEP editors fine with sorting the list in this PR?

Reply:
Do you also want to sort the list of authors?

Reply:
Sorted the list of members in 58d269b32a3b144b4854a7b21105d586ef441a99.

Reply:
Marking this conversation as resolved, since the list is now sorted.